daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -208,7 +208,12 @@ function Change-InstanceProperties {
   # Change time zone to Coordinated Universal Time.
   Run-Command tzutil /s 'UTC'
 
-  Run-Command powercfg /hibernate off
+  # Not supported on 6.1 client, but is supported on 6.1 server
+  $pn_path = 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion'
+  $pn = (Get-ItemProperty -Path $pn_path -Name ProductName).ProductName
+  if (!($pn -like '*Windows 7*')) {
```

Comment:
if ($pn -notlike '*Windows 7*')

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-6

Reply:
Updated to notlike comparison, left wildcards around product name to capture multiple versions of Windows 7.